ldes-server-maintenance/ldes-server-compaction/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/compaction/batch/CompactionDbWriter.java

```diff
@@ -12,27 +13,30 @@
 import java.util.Set;
 
 @Component
-public class PaginationCompactionService {
+public class CompactionDbWriter {
```

Comment:
Personally (so this does not mean you SHOULD), I would leave out the "DB". Writing implies already a datasource interaction